src/components/SuperDashListCard/SuperDashListCard.tsx

```diff
@@ -23,7 +23,6 @@ function SuperDashListCard(props: SuperDashListCardProps): JSX.Element {
 
   function handleClick() {
     const url = '/orgdash/id=' + props.id;
-    window.location.replace(url);
```

Comment:
@adepeju4 

Thanks for contributing!

Please take a look at the application code, if anything is depending upon the line you removed.
Almost every admin page is depending on it.

Reply:
Okayyy. I had no idea such a thing was happening. I think I would leave it as it is instead and close this PR

Reply:
1. @adepeju4 how can we test for this functionality so that the tests fail if this line is removed?
1. How can we prevent others from removing this code in future? What warnings can we give?

Closing the PR will not fix the problem. Make the appropriate adjustments.